### PR TITLE
[BROWSEUI] Load settings on CInternetToolbar::OnContextMenu

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose

Based on KRosUser's `go.patch`.
JIRA issue: [CORE-19145](https://jira.reactos.org/browse/CORE-19145)

## TODO

- [x] Do tests.

### CODE REVIEW

**Review by JoachimHenze — APPROVED** [review]
Approved. Let's wait for a 2nd approval, possible that someone thinks it should be placed at another position instead. If not, I am fine with it.

File: dll/win32/browseui/internettoolbar.cpp
Comment (HBelusca) [inline dll/win32/browseui/internettoolbar.cpp:1719]: I think the question of @JoachimHenze is like: "why putting this loading just in the middle of code that deals with setting up the menus, instead of before, or after?"
(and I would agree with this questioning)

Comment (katahiromz) [reply dll/win32/browseui/internettoolbar.cpp]: Moved.

**Review by HBelusca — APPROVED** [review]

**Pull request merged**
